libs/coin-modules/coin-mina/src/test/synchronisation.unit.test.ts

```diff
@@ -0,0 +1,30 @@
+import invariant from "invariant";
+import { getTransactions } from "../api";
+import { mapRosettaTxnToOperation } from "../bridge/synchronisation";
+import { setConfig } from "./testUtils";
+
+setConfig();
+
+describe("synchronisation", () => {
+  it("should map rosetta txn to operation", async () => {
+    const txns = await getTransactions("B62qjWLs1W3J2fFGixeX49w1o7VvSGuMBNotnFhzs3PZ7PbtdFbhdeD");
```

Comment:
No real backend request in unit test please 🙏 

You need to create a .integ.test.ts if it calls a real backend